src/main/kotlin/domain/LottoGenerator.kt

```diff
@@ -0,0 +1,14 @@
+package domain
+
+class LottoGenerator() : LottoGeneratorInterface {
+    override fun generateLotto(): Lotto {
+        val possibleLottoNumbers = (MINIMUM_LOTTO_NUMBER..MAXIMUM_LOTTO_NUMBER).toList()
+        return Lotto(possibleLottoNumbers.map { LottoNumber(it) }.shuffled().take(LOTTO_NUMBER_COUNT))
```

Comment:
```suggestion
        val possibleLottoNumbers = MINIMUM_LOTTO_NUMBER..MAXIMUM_LOTTO_NUMBER
        return Lotto(possibleLottoNumbers.map { LottoNumber(it) }.shuffled().take(LOTTO_NUMBER_COUNT))
```
모든 컬렉션과 관련된 함수들(map, shuffled, take 등)은 매번 복사본을 만들게 됩니다.
따라서, toList 함수를 호출하지 않으셔도 됩니다.